mopidy_beets/translator.py

```diff
@@ -80,16 +82,17 @@ def parse_album(data, api):
     # The order of items is based on the above documentation.
     # Attributes without corresponding Beets data are mapped to 'None'.
     mapping = {
-        'uri': lambda d: assemble_uri('beets:library:album', id_value=d['id']),
-        'name': 'album',
-        'artists': lambda d: _filter_none([parse_artist(d, 'albumartist')]),
-        'num_tracks': 'tracktotal',
-        'num_discs': 'disctotal',
-        'date': lambda d: parse_date(d),
-        'musicbrainz_id': 'mb_albumid',
+        "uri": lambda d: assemble_uri("beets:library:album", id_value=d["id"]),
+        "name": "album",
+        "artists": lambda d: _filter_none([parse_artist(d, "albumartist")]),
+        "num_tracks": "tracktotal",
+        "num_discs": "disctotal",
+        "date": lambda d: parse_date(d),
+        "musicbrainz_id": "mb_albumid",
         # TODO: 'images' is deprecated since v1.2 - move to Library.get_images
-        'images': lambda d, api=api: _filter_none(
-            [api.get_album_art_url(d['id'])]),
+        "images": lambda d, api=api: _filter_none(
+            [api.get_album_art_url(d["id"])]
+        ),
```

Comment:
Support for `Album.images` is removed in Mopidy 3.0, so this has to be removed to be able to create `Album` instances now.

Reply:
Good - I will remove it.
I did not find a reference to a functional replacement. Are images handled differently now?

Thank you for your helpful review!

Reply:
https://docs.mopidy.com/en/latest/changelog/#models